ctru-rs/src/services/cam.rs

```diff
@@ -21,191 +20,165 @@ pub struct Cam {
     pub both_outer_cams: BothOutwardCam,
 }
 
-bitflags! {
-    /// A set of flags to be passed to [Camera::flip_image]
-    #[derive(Default)]
-    pub struct CamFlip: u32 {
-        const NONE       = ctru_sys::FLIP_NONE;
-        const HORIZONTAL = ctru_sys::FLIP_HORIZONTAL;
-        const VERTICAL   = ctru_sys::FLIP_VERTICAL;
-        const REVERSE    = ctru_sys::FLIP_REVERSE;
-    }
+/// Flag to pass to [Camera::flip_image]
```

Comment:
tiny nit, I usually prefer ``[`backticks`]`` inside the doc links for stuff that refers to real APIs like this. I guess you didn't change this but just something to think about while cleaning up docstrings.